plugins/woocommerce-admin/client/marketplace/components/woo-connect-plugin/woo-connect-plugin.scss

```
@@ -0,0 +1,61 @@
+@import '@wordpress/base-styles/_colors.native.scss';
+@import '../../stylesheets/_variables.scss';
+
+.woocommerce-marketplace__woo-connect-plugin__notices {
+	.components-notice {
+		margin-left: 0;
+		margin-right: 0;
+		background-color: #fff;
+		box-shadow: 0 2px 6px 0 rgba($gray-100, 0.05);
+		border: 1px solid var(--gutenberg-gray-100, #f0f0f0);
+		padding-right: $grid-unit-15;
+		position: relative;
+
+		&::before {
+			content: '';
+			display: block;
+			width: 4px;
+			height: 100%;
+			background-color: var(--wp-admin-theme-color, #007cba);
+			position: absolute;
+			left: 0;
+			top: 0;
+			bottom: 0;
+		}
+
+		&.is-error::before {
+			background-color: $alert-red;
+		}
+
+		.components-notice__content {
+			align-items: center;
+			gap: $grid-unit-15;
+			padding: 0px;
+
+			.components-notice__buttons {
+				margin-top: 12px;
+			}
+		}
```

Shouldn't it be like this?
```suggestion
		.components-notice__content {
			align-items: center;
			gap: $grid-unit-15;
			padding: 0px;
		}
		
		.components-notice__buttons {
			margin-top: 12px;
		}
```

---

Fixed with https://github.com/woocommerce/woocommerce/pull/44279/commits/fca09b500fbbc61683473a2270afb490739c36e8